src/app/Traits/InCents.php

```diff
@@ -44,6 +39,23 @@ public function inCents(bool $mode = true)
 
         $this->inCents = $mode;
 
+        $this->convertCentValues();
+
+        return $this;
+    }
+
+    private function validate(): void
```

Comment:
revert here. Whenever we use model traits we want the minimum amount of methods or at least very specific method names to avoid colliding with model methods.